package.json

```diff
@@ -36,6 +36,7 @@
     "packages/*"
   ],
   "scripts": {
+    "create-screen": "node scripts/showtime-cli",
```

Comment:
[experimental] trying to make a cli to create screens. Need some work but basically you can run yarn create-screen [SCREEN Name] and it'll create folders/file for you

Reply:
wow, really needed this!